sdk/include/cdefs.h

```diff
@@ -94,7 +94,7 @@ using _Bool = bool;
  * other places) to avoid the namespace pollution from including `string.h` but
  * is either constant folded in the front end or expanded to a libcall.
  */
-unsigned __builtin_strlen(const char *str) __asm__("_Z6strlenPKc");
+unsigned __builtin_strlen(const char *str);
```

Comment:
I believe you can remove this declaration entirely.  Without it, clang will use the built-in definition.